homeassistant/components/google_assistant/helpers.py

```diff
@@ -372,14 +372,19 @@ def is_supported(self) -> bool:
     @callback
     def might_2fa(self) -> bool:
```

Comment:
Is it correct that `should_expose_local` should still depend on `might_2fa`?

Reply:
Yes. We need the final decision if it might_2fa to decide if we want to expose devices.